integrations/stripe/src/definitions/channels.ts

```diff
@@ -0,0 +1,7 @@
+import { IntegrationDefinitionProps, messages } from '@botpress/sdk'
+
+export const channels = {
+  channel: {
+    messages: { ...messages.defaults },
```

Comment:
**issue:** I don't believe stripe exposes a messaging channel. It should be removed.